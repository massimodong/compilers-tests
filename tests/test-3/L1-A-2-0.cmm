float sIhruYtW9Z1 

int ;
