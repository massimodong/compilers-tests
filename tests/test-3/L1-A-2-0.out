Error Type B at Line 3
